src/lib/modules/Document/DocumentCard/DocumentCard.js

```diff
@@ -46,6 +46,11 @@ class DocumentCard extends Component {
     if (data.metadata.other_authors) {
       authors = authors + 'et al';
     }
+    const volume =
+      metadata.relations &&
+      metadata.relations.multipart_monograph &&
+      metadata.relations.multipart_monograph[0].volume;
```

Comment:
could this be replaced with something like
```suggestion
    const volume = _get(metadata, 'relations.multipart_monograph[0].volume')
```
(_get from lodash)